tests/test_docs/sample/test_shoulder_create_ark_NULL.sample

```diff
@@ -16,8 +16,8 @@ at [MEMORY-ADDRESS]>,
   'name': '91101/r01 test org',
   'prefix': 'ark:/99920/',
   'prefix_shares_datacenter': False,
-  'registration_agency': None,
-  'shoulder_type': None,
+  'registration_agency': <RegistrationAgency: RegistrationAgency object (1)>,
```

Comment:
Is "<RegistrationAgency: RegistrationAgency object (1)>" just a statement, or it refers to a value in the data model?

Reply:
It is the stringified value in the data model that is used for comparison in the test case. See for example the tests in test_shoulder_ark.py wherein calls like `sample.assert_match(tests.util.util.shoulder_to_dict(s), 'basic')` are being made to compare the actual with expected results.